### PR TITLE
RCT unpacker plus DQM 75X

### PR DESCRIPTION
**Pull request by nsmith-**
This is a 75X copy of #10381
A PR for the subset for 76X that hasn't been merged already
(namely standard sequence + DQM updates)
is #10658

### CODE REVIEW

**Comment by deguio** [comment]
please test

File: DQM/L1TMonitor/src/L1TRCT.cc [hunk 1]
```diff
@@ -176,35 +213,63 @@ void L1TRCT::analyze(const Event & e, const EventSetup & c)
...
+    doHdLayer2 = false;
+  }
+
+  if (!emLayer2.isValid()) {
+    edm::LogInfo("DataNotFound") << "can't find L1CaloEmCollection - Layer2";
+    doEmLayer2 = false;
+  }
+
+
+  std::cout<<doHd<<doEm<<doHdLayer2<<doEmLayer2<<std::endl;
```
Comment (deguio) [inline DQM/L1TMonitor/src/L1TRCT.cc]: please remove cout

**Comment by deguio** [comment]
-1
apart from the cout to be removed, the PR looks ok for what concerns DQM and can go in.
rejecting for now, but feel free to update this branch and the PR will be reconsidered.

**Comment by deguio** [comment]
please test

**Comment by deguio** [comment]
+1
provided it passes the tests

**Comment by danduggan** [comment]
-1
still needs another commit before ready to go in

**Comment by danduggan** [comment]
please test

**Comment by nsmith-** [comment]
Superceded by #10881

**Pull request closed without merging**